app/react/portainer/registries/ListView/DefaultRegistry/DefaultRegistryAction.tsx

```diff
@@ -29,7 +29,9 @@ export function DefaultRegistryAction() {
       {!hideDefaultRegistry ? (
         <div className="vertical-center">
           <Button
-            className="btn btn-xs btn-light vertical-center"
+            color="danger"
+            size="xsmall"
+            className="vertical-center"
```

Comment:
we can actually remove the Icon component and pass `icon={EyeOff}` instead. then the class is not needed

Reply:
same for L50

Reply:
did you push this change?

Reply:
this is available on EE, but you didn't push for CE